www/app/app.module.js

```diff
@@ -1,13 +1,15 @@
 (function() {
   'use strict';
 
-  // Budget Rent a Car App
-
-  angular.module('budgetrentacar', [
+  angular
+    .module('budgetrentacar', [
       'ionic',
+      'firebase',
+      'ngStorage',
+      'angular-jwt',
       'ui.router',
-      'budgetrentacar.home',
-      'budgetrentacar.form'
+   
+      'budgetrentacar.login',
+      'budgetrentacar.home'
     ]);
-
 })();
```

Comment:
Try not to repeat this in the future, git itself is telling us that there needs to be a newline at the end of EVERY FILE.